src/snake.rs

```diff
@@ -47,14 +38,14 @@ impl Snake {
     }
 
     pub fn on_game_starting(&self, _: &GameStarting) {
-
+        debug!(target: LOG_TARGET, "All snakes are ready to rock. Game is starting.");
     }
 
     pub fn on_player_registered(&self, _: &PlayerRegistered) {
-
+        debug!(target: LOG_TARGET, "Player has been registered.");
     }
 
     pub fn on_invalid_playername(&self, _: &InvalidPlayerName) {
-
+        debug!(target: LOG_TARGET, "I'm sorry, I cannot let you do that Dave.");
```

Comment:
Since this is in a part of the program the end user should tinker with, I think it's alright to have corny error messages, but I will change it to something reasonable instead. :)